spec/agent.js

```diff
@@ -0,0 +1,74 @@
+"use strict";
+
+const http = require("http");
+const bodyParser = require("body-parser");
+const express = require("express");
+const Int64BE = require("int64-buffer").Int64BE;
+
+function flat(array) {
+  const flat = [];
+  array.forEach((inner) => {
+    inner.forEach((item) => {
+      flat.push(item);
+    });
+  });
+  return flat;
+}
+
+const zipkinV2toDD = (trace) => {
```

Comment:
Since zipkin is in json I wonder if this combined test utilities approach is necessary.  I also suspect there are licensing issues we'd want to avoid (lambda-nodejs is without license).

Reply:
Don't think there should be licensing issues as _I think_ this code is original and just deals with dd formatted data but good point about it being JSON already. Will update.